proxy/operator_resource/resource_manager.py

```diff
@@ -270,6 +328,8 @@ async def _activate_signer_list(self) -> None:
                     if op_signer := self._disabled_signer_dict.pop(op_signer.owner, None):
                         # move the key to the active list for using in tx processing
                         self._active_signer_dict[op_signer.owner] = op_signer
+                        await self._send_op_resource_holder_stats(op_signer)
```

Comment:
why is `await` here?

Reply:
The signer can be active, but he can have disabled holders.
So, move this call to the up level.

Reply:
Ah, got it. This is fixed.